src/main/java/graphql/servlet/context/DefaultGraphQLContextBuilder.java

```diff
@@ -0,0 +1,30 @@
+package graphql.servlet.context;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.Part;
+import javax.websocket.Session;
+import javax.websocket.server.HandshakeRequest;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Returns an empty context.
+ */
+public class DefaultGraphQLContextBuilder implements GraphQLContextBuilder {
+
+    @Override
+    public GraphQLContext build(HttpServletRequest httpServletRequest, HttpServletResponse httpServletResponse, Map<String, List<Part>> fileParts) {
```

Comment:
I see we're passing this `fileParts` all the way down to this builder after we've extracted them with some heavy logic within the `AbstractGraphQLHttpServlet`. We should extract that part into its own class for starters, but also: we might as well push the extraction all the way down to the builder where we construct the actual context. Seeing that the `HttpServletRequest` that we retrieve the parts from is passed down as well anyway. That would clean up a lot of intermediate methods that now have to pass along that `fileParts` parameter.